### PR TITLE
Logout test for nav bar test

### PR DESCRIPTION
**Pull request by norisalsaadie**
Following libero/reviewer#856

### CODE REVIEW

File: tests/browser-tests/Login.browser.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { DashboardPage, FilesPage, LoginPage, AuthorDetailsPage, DetailsPage, Na
 
 fixture`Getting Started`.page`http://localhost:9000`;
 
-test('assert nav bar', async() => {
+test.only('assert nav bar', async() => {
```
Comment (diversemix) [inline tests/browser-tests/Login.browser.ts:5]: this left in by mistake? (not that this file is up to scratch yet anyhow)

Comment (norisalsaadie) [reply tests/browser-tests/Login.browser.ts:5]: It needs a better name, but it's literally a test that focus on clicking the nav etc

Comment (norisalsaadie) [reply tests/browser-tests/Login.browser.ts:5]: Ohh. opps. Nice catch with the .only

Comment (norisalsaadie) [reply tests/browser-tests/Login.browser.ts]: Pushed up a fix.

**Pull request merged**